### PR TITLE
 Video effects conceptual docs in storybook

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
 Video effects conceptual docs in storybook

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/VideoEffects.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+/>
+
+## Blurred and Custom Backgrounds
+
+<DetailedBetaBanner version={'1.5.1-beta.5'} />
+
+Azure Communication Services UI Library is adding support for Blurred and Custom Backgrounds.
+With this new feature, users can blur the user background or change the background to a provided default image gallery option.
+Contoso can set their own custom images that the end user can then set as their background, allowing for greater customization of the calling experience.
+Enabling this feature will enable greater privacy and allow for greater focus on the person by removing the existing background behind the user.
```
Comment (edwardlee-msft) [inline packages/storybook/stories/VideoEffects.stories.mdx:19]:
```suggestion
Enable this feature to enhance privacy and to also direct focus on the individual by removing their existing  "real" background with a custom background or effect.
```

Feel free to rephrase "real". I was trying to define a difference between a custom and a background from the actual camera.

File: packages/storybook/stories/VideoEffects.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+**Personalization:** Custom backgrounds can be used to express individuality and personal preferences, adding a fun and creative element to video calls.
+
+### Incorporating Blurred and Customer backgrounds into your experience
+
+The UI Library enables users to enhance their video conferencing experiences by providing the option to use blurred and custom backgrounds within the CallComposite and CallWithChat experiences.
+Blurred and custom backgrounds can be enabled both in Mobile Web sessions and in Desktop Web sessions.
+Blurred and Custom Backgrounds are enabled by default and are automatically included within the CallComposite and CallWithChatComposite experiences.
+
+Users can select if they want blurred, custom, or no backgrounds from the start screen or in-call, in the camera and effects settings.
```
Comment (edwardlee-msft) [inline packages/storybook/stories/VideoEffects.stories.mdx:34]:
```suggestion
Users can select if they want blurred, custom, or no backgrounds from the start screen or in-call though the camera and effects settings.
```

File: packages/storybook/stories/VideoEffects.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Blurred and Custom Backgrounds
+
+<DetailedBetaBanner version={'1.5.1-beta.5'} />
+
+Azure Communication Services UI Library is adding support for Blurred and Custom Backgrounds.
+With this new feature, users can blur the user background or change the background to a provided default image gallery option.
+Contoso can set their own custom images that the end user can then set as their background, allowing for greater customization of the calling experience.
+Enabling this feature will enable greater privacy and allow for greater focus on the person by removing the existing background behind the user.
+Here are the main scenarios where blurred and custom backgrounds are useful:
```
Comment (mgamis-msft) [inline packages/storybook/stories/VideoEffects.stories.mdx:20]: I would rephrase this as 'The main scenarios where blurred and custom backgrounds are useful are:'

File: packages/storybook/public/images/videoeffects-call-button.png
Comment (mgamis-msft) [inline packages/storybook/public/images/videoeffects-call-button.png:1]: nit: Maybe put a mouse cursor over Effects button to highlight that the user should click it. Or you can add  a short description in the 'Selecting background effect in call' section

**Comment by mgamis-msft** [comment]
Looks great overall!

File: change-beta/@azure-communication-react-eb0a52f5-c4eb-4da2-94d9-33fec1ebd417.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```
Comment (mgamis-msft) [inline change-beta/@azure-communication-react-eb0a52f5-c4eb-4da2-94d9-33fec1ebd417.json:2]: I believe storybook changes should be set as none

**Review by edwardlee-msft — APPROVED** [review]

**Review by carocao-msft — APPROVED** [review]
